rust/operator-binary/src/authorization/opa.rs

```diff
@@ -19,6 +19,8 @@ pub struct TrinoOpaConfig {
     opa_authorizer_name: String,
     non_batched_connection_string: String,
     batched_connection_string: Option<String>,
+    row_filters_connection_string: Option<String>,
+    column_masking_connection_string: Option<String>,
     allow_permission_management_operations: bool,
```

Comment:
Can you add more documentation to this struct? Are these settings that are set in the trino config (I guess they are) - where are these connection strings pointing to? Specific opa endpoints?